src/video/video.c

```diff
@@ -169,12 +169,17 @@ static void video_set_blend_mode(object_array_blend_mode request_mode) {
 }
 
 void video_area_capture(surface *sur, int x, int y, int w, int h) {
+    // XXX hack: round up read width to nearest multiple of 4 for glReadPixels call
+    // could someone who knows GL better replace this magic number 4 with something from the API?
+    int bufw = (w + 3) / 4 * 4;
```

Comment:
Why this ?

Reply:
On my computer (NVidia, Windows), calling glGetPixels with widths that aren't multiples of 4 produces output with weird stride:
![image](https://github.com/user-attachments/assets/816c06f0-9a4b-4677-834e-ca9a040797dd)

Reply:
I see. Try setting glPixelStorei(GL_PACK_ALIGNMENT, 1);

Reply:
that does it, thanks!